src/transformers/modeling_tf_utils.py

```diff
@@ -47,7 +47,7 @@ def num_parameters(self, only_trainable: bool = False) -> int:
             return self.count_params()
 
 
-class TFPreTrainedModel(tf.keras.Model, TFModelUtils):
+class TFPreTrainedModel(tf.keras.Model, TFModelUtilsMixin):
```

Comment:
@aaugustin ^^ Is this the right order or should it be `TFPreTrainedModel(XxxMixin, YyySuperClass)`?

Reply:
Both are fine, as long as you get the method resolution order you expect.

Often, mixins add new methods but don't override existing methods, and they're added at the end of the inheritance chain like you did.